meeting-notes/2020-02-12_pipeline-dev-meeting.md

```diff
@@ -0,0 +1,71 @@
+# Dashboards & monitoring
+
+## Meeting
+
+2020-02-12
+
+Present: Max, Emily, Julian
+
+#### Julian's Notes
+
+- Monitoring & logging hard to stay on stop with at scale
+- Easier to just retry & make transformers idempotent?
+
+- [ ] Talk to Eric/David about whether we need to tar the products/output of
+Makeflow steps (in addition to raw input data)
+
+- Look at TimescaleDB
+    - Ideally store plaintext metadata with files so we can regenerate database
+
+- Catcher's Mitt: Thing that triggers processing/tarring
+- Pull data to ds06
+    - Globus - going away soon because NCSA
+     will not be processing any more data and are trying to get the data off
+     their servers. (According to Max)
+- How to know a scan is done?
```

Comment:
Jeffrey has a scan log CSV file that could be pushed to the gantry cache server via FTP.